Remotion/Mixins/Samples/DynamicMixinBuilding/UnitTests/DynamicMixinBuilderTest.cs

```diff
@@ -62,11 +62,12 @@ public static T Create<T> (ParamList constructorParameters)
     private MethodInvocationHandler _invocationHandler;
     private DynamicMixinBuilder _builder;
     private ServiceLocatorScope _serviceLocatorScope;
+    private string _directory;
```

Comment:
Since we called the variable during creation `tempDirectory` I think we should also call this `_tempDirectory`. This makes the use case at least a bit clearer.